Source/Core/Core/GeckoCodeConfig.cpp

```diff
@@ -23,7 +23,12 @@ std::vector<GeckoCode> DownloadCodes(std::string gametdb_id, bool* succeeded, bo
   const std::string protocol = use_https ? "https://" : "http://";
 
   // codes.rc24.xyz is a mirror of the now defunct geckocodes.org.
-  std::string endpoint{protocol + "codes.rc24.xyz/txt.php?txt=" + gametdb_id};
+  // If getting Mario Kart Wii codes, pull them from mkwtas.com.
+  std::string endpoint{protocol};
+  if (gametdb_id.substr(0, 3) == "RMC")
+    endpoint += "mkwtas.com/api/" + gametdb_id + ".txt";
```

Comment:
Instead of an if/else, can we send `gametdb_id.substr(0, 3)` to a function that gets the endpoint? This would allow other communities to PR their endpoints, for example.